src/client/firefox/create.js

```diff
@@ -31,12 +32,16 @@ export function createFrame(frame: FramePacket): Frame {
   };
 }
 
-export function createSource(source: SourcePayload): Source {
+export function createSource(
+  source: SourcePayload,
+  debuggerClientTraits: DebuggerClientTraits
```

Comment:
I would prefer passing a smaller options object in: `{ supportsWasm: bool }`. This will keep the interface smaller and more flexible